var/spack/repos/builtin/packages/openspeedshop/package.py

```diff
@@ -83,8 +77,8 @@ class Openspeedshop(Package):
                          to point to target build.")
     variant('cuda', default=False,
             description="build with cuda packages included.")
-    variant('ptgf', default=False,
-            description="build with the PTGF based gui package enabled.")
+    variant('noqt3gui', default=False,
+            description="build without the build Qt3 gui package enabled.")
```

Comment:
I think it makes more sense to use positive variants. Otherwise you end up with a double-negative. `~noqt3qui` means "don't build without the GUI". You can change the default to True if you want.